src/OrchardCore.Cms.Web/Startup.cs

```diff
@@ -1,26 +1,7 @@
-using Microsoft.AspNetCore.Builder;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Hosting;
+namespace OrchardCore.Cms.Web;
 
-namespace OrchardCore.Cms.Web
+// Still used in Tests
+public class Startup
```

Comment:
Could you updatre the tests to not require this class https://docs.microsoft.com/en-us/aspnet/core/test/integration-tests?view=aspnetcore-6.0

Reply:
I don't know how we can get rid of it.

Reply:
fixled